### DISCUSSION TITLE
add jukebox shortcuts

### DISCUSSION THREAD

**Discussion by ampersand64** [ppy/osu#30329] [Ideas] [answered]
Stable's main-menu had convenient keyboard shortcuts to pause/play the jukebox music, shuffle, play next song, or stop playback. 
Adding keyboard controls for the main-menu jukebox is a small but easy quality of life improvement to Lazer.

**Comment by bdach** [comment, marked as answer]
Keyboard controls for music playback already exist though? By default it's <kbd>F1</kbd> for previous, <kbd>F3</kbd> for play/pause and <kbd>F5</kbd> for next, all of which match stable. <kbd>F2</kbd> and <kbd>F4</kbd> are not there but they feel kinda pointless, and I'm not aware of a "shuffle" shortcut on stable?

**Reply by ampersand64** [reply]
my sincerest apologies for the error.
